specs/protocol/rollup-node-p2p.md

```diff
@@ -403,6 +416,8 @@ Implementations may opt for a different limit, since this sync method is optiona
   matching the `ExecutionPayload` SSZ definition of the L1 Merge, L2 Bedrock and L2 Regolith, L2 Canyon versions.
 - `1`: SSZ-encoded `ExecutionPayloadEnvelope` with Snappy framing compression,
   matching the `ExecutionPayloadEnvelope` SSZ definition of the L2 Ecotone version.
+- `2`: SSZ-encoded `ExecutionPayload` with Snappy framing compression,
```

Comment:
From the proposed changes [here](https://github.com/ethereum-optimism/op-geth/pull/383/files#diff-50e0dca0bd0ac667ca6a4a3b762c9346030f346fdd9b48f3f83be675a43423eaR91-R94) and as further elaborated [here](https://github.com/ethereum-optimism/optimism/issues/12044#issuecomment-2369618003), the `ExecutionPayload` (and not `ExecutionPayloadEnvelope`) will be extended with `withdrawalsRoot`